pandas/tests/frame/test_api.py

```diff
@@ -356,7 +356,7 @@ def test_axis_aliases(self):
         assert_series_equal(result, expected)
 
     def test_class_axis(self):
-        # https://github.com/pandas-dev/pandas/issues/18147
+        # GH 18147
```

Comment:
Generally still ask that you try to *minimize* your changes as much as possible. Obviously changing comments isn't going to affect any of the code base but it still makes the diffs of your PRs unnecessarily larger, which makes reviews more difficult.

You should always look for logical separation points for smaller PRs, i.e. if you really want to clean up these comments then make a separate PR for that rather than bundling here. For this change please revert this file to master

Reply:
IMO these are trivial changes that don't get done otherwise, and what's more I did get asked to do this in several PRs, so I just do it preemptively.

However, I separated the commits between fixturizing and import-cleaning, so you can review the separate commits (almost) as separate PRs.